internal/output/json.go

```diff
@@ -92,8 +99,22 @@ func (f *JSON) Error(value interface{}) {
 
 	_, err = f.cfg.OutWriter.Write(b)
 	if err != nil {
-		multilog.Error("Could not write json output, error: %v", err)
+		if isPipeClosedError(err) {
+			logging.Error("Could not write json output, error: %v", err) // do not log to rollbar
+		} else {
+			multilog.Error("Could not write json output, error: %v", err)
+		}
+	}
+}
+
+func isPipeClosedError(err error) bool {
+	pipeErr := errors.Is(err, syscall.EPIPE)
+	if runtime.GOOS == "windows" && errors.Is(err, syscall.Errno(242)) {
+		// Note: 232 is Windows error code ERROR_NO_DATA, "The pipe is being closed".
```

Comment:
Do we know if the Go library uses error code `109` as well? It is `ERROR_BROKEN_PIPE`

https://learn.microsoft.com/en-us/windows/win32/debug/system-error-codes--0-499-

Reply:
I don't know. Regardless, that error message is "The pipe has been ended." As mentioned previously, I think we want to be as narrow as possible right now when turning off rollbar logging. I haven't seen that error message in rollbar yet.